### DISCUSSION TITLE
手机浏览器edge 编辑文本显示异常

### DISCUSSION THREAD

**Discussion by meng-plus** [1Panel-dev/1Panel#5406] [Q&A]
### 1Panel 版本

10版

### 问题描述

手机edge 桌面版 打开文本编辑修改时 修改区域被强制放最大，切无法缩小 导致无法正常看到前后文修改

### 重现步骤

手机浏览器edge 桌面版打开
随便打开个文本，进入编辑后迅速最大化文本区，导致无法正常看到前后文进行编辑，也无法双指缩放显示
![Screenshot_2024-06-11-02-42-04-83_df198e732186825c8df26e3c5a10d7cd](https://github.com/1Panel-dev/1Panel/assets/37764731/dba421e1-de96-4a7b-a193-00985cd9eaa1)

手机版模式还算正常
![Screenshot_2024-06-11-02-35-51-89_df198e732186825c8df26e3c5a10d7cd](https://github.com/1Panel-dev/1Panel/assets/37764731/383c0645-8435-440e-941a-f436d021a433)

### 期待的正确结果

编辑功能不调整网页缩放
调用输入法不自动缩放网页

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，由于目前精力有限，我们暂时无法完成上述优化，但会在后续版本中考虑。
